tests/integration/test_grant_and_revoke/test.py

```diff
@@ -377,6 +377,8 @@ def test_introspection():
     instance.query("GRANT SELECT ON test.table TO A")
     instance.query("GRANT CREATE ON *.* TO B WITH GRANT OPTION")
 
+    all_access_except_show_named_collections = "SHOW, SELECT, INSERT, ALTER, CREATE, DROP, TRUNCATE, OPTIMIZE, BACKUP, KILL QUERY, KILL TRANSACTION, MOVE PARTITION BETWEEN SHARDS, CREATE USER, ALTER USER, DROP USER, CREATE ROLE, ALTER ROLE, DROP ROLE, ROLE ADMIN, CREATE ROW POLICY, ALTER ROW POLICY, DROP ROW POLICY, CREATE QUOTA, ALTER QUOTA, DROP QUOTA, CREATE SETTINGS PROFILE, ALTER SETTINGS PROFILE, DROP SETTINGS PROFILE, SHOW ACCESS, SYSTEM, dictGet, INTROSPECTION, SOURCES, CLUSTER"
```

Comment:
why are we not adding `show_named_collections` access to this test ?

Reply:
Because I updated the test instead of updating the config. But indeed I'd better update config as the test will be easier to read.